src/layer.rs

```diff
@@ -480,6 +454,20 @@ impl Default for Layer {
     }
 }
 
+/// [UFO layer info], corresponding to a 'layerinfo.plist' file in a layer directory.
+///
+/// This will be merged into a [`Layer`].
+///
+/// [UFO layer info]: https://unifiedfontobject.org/versions/ufo3/glyphs/layerinfo.plist/
+#[derive(Debug, Clone, Deserialize)]
+struct LayerInfo {
+    /// Color field.
```

Comment:
there is no obligation to document a private type, and while I think it is useful to do this much of the time in this case it does not feel helpful.